install.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# TODO(everyone): Keep this script simple and easily auditable.
+
+set -e
+
+if [ "$(uname -m)" != "x86_64" ]; then
+  echo "Error: Unsupported architecture $(uname -m). Only x64 binaries are available." 1>&2
+  exit 1
+fi
+
+if [ "$OS" = "Windows_NT" ]; then
+  target="windows-386"
+  ext=".zip"
+  if ! command -v unzip >/dev/null; then
+    echo "Error: unzip is required to install coder-cli" 1>&2
+    exit 1
+  fi
+else
+  if ! command -v tar >/dev/null; then
+    echo "Error: tar is required to install coder-cli" 1>&2
+    exit 1
+  fi
+  ext=".tar.gz"
+  case $(uname -s) in
+  Darwin) target="darwin-amd64" ;;
+  *) target="linux-amd64" ;;
+  esac
+fi
+
+if [ $# -eq 0 ]; then
+  coder_asset_path=$(
+    curl -sSf https://github.com/cdr/coder-cli/releases |
+      grep -o "/cdr/coder-cli/releases/download/.*/coder-cli-${target}${ext}" |
+      head -n 1
+  )
+  if [ ! "$coder_asset_path" ]; then
+    echo "Error: Unable to find latest coder-cli release on GitHub." 1>&2
+    exit 1
+  fi
+  cdr_uri="https://github.com${coder_asset_path}"
+else
+  cdr_uri="https://github.com/cdr/coder-cli/releases/download/${1}/coder-cli-${target}${ext}"
+fi
+
+coder_install="${CODER_INSTALL:-$HOME/.coder}"
+bin_dir="$coder_install/bin"
+exe="$bin_dir/coder"
+
+if [ ! -d "$bin_dir" ]; then
+  mkdir -p "$bin_dir"
+fi
+
+curl --fail --location --progress-bar --output "$exe$ext" "$cdr_uri"
```

Comment:
Can we make the flag short here too?

Reply:
Why though? Not sure why we'd intentionally make it harder to read 😆

Reply:
No one uses the long flags for `curl`, everyone knows the short flags so they're easier to read.

Reply:
I do not know the long flags and I appreciate that moogster went long.

Reply:
Same here, I pretty much always use long flags wherever possible

Reply:
Then let's be consistent with the above invocation of curl.